fury/ui/core.py

```diff
@@ -315,7 +315,7 @@ def handle_events(self, actor):
         """
         actor.add_event_handler(self.mouse_button_down_callback, "pointer_down")
         actor.add_event_handler(self.mouse_button_up_callback, "pointer_up")
-        actor.add_event_handler(self.mouse_move_callback, "pointer_move")
+        actor.add_event_handler(self.mouse_move_callback, "pointer_drag")
```

Comment:
Lets also use the `EventType.POINTER_DRAG` and instead of hard coding as we did in `window.py`

Reply:
You mean `EventType.POINTER_DRAG` and not `EventType.POINTER_DOWN`.

Reply:
I meant using the below things instead of harcoded values

```
actor.add_event_handler(self.mouse_button_down_callback, EventType.POINTER_DOWN)
 actor.add_event_handler(self.mouse_button_up_callback, EventType.POINTER_UP)
 actor.add_event_handler(self.mouse_move_callback, EventType.POINTER_DRAG)
```